src/js/components/AppListStatusFilterComponent.jsx

```diff
@@ -33,18 +26,18 @@ var AppListStatusFilterComponent = React.createClass({
 
   getInitialState: function () {
     return {
-      appsStatusesCount: getInitialAppStatusCount(),
+      appsStatusesCount: {},
```

Comment:
OK let's keep `appsStatusesCount`. But – nitpick of the nitpick – keep in mind `status` is also plural in English (hence the confusion) ;)